src/config/config.hpp

```diff
@@ -39,6 +40,18 @@ struct RemoteIAM {
     UtilsTime::Duration mRequestTimeout;
 };
 
+/*
+ * PKCS11ModuleParams PKCS11 module parameters.
```

Comment:
No need to put class/struct name in comment.  Same for other comments in this file.